centaur/src/main/resources/integrationTestCases/somatic_cnv_pair.test

```diff
@@ -1,6 +1,6 @@
 name: cnv_somatic_pair
 testFormat: workflowsuccess
-
+ignore: true
```

Comment:
curious as to why this is now being ignored?

Reply:
The workflow is not completing properly and needs to be updated. @ruchim will be making the updates for CNV and Mutect workflows.

Reply:
Could @ruchim please confirm her end of this bargain?

Reply:
currently my open PR includes changes to this workflow, so it will bring it back right away :)